### ISSUE TITLE
gray scale image issue

### ISSUE THREAD

**Issue by yaozr** [YuzhouPeng/unet-tensorrt#10]
for the genwts.py:

tmp = torch.ones(1, 3, 224, 224).to('cuda:0')
    print('input: ', tmp)
    out = net(tmp)
    print('output:', out)
    summary(net, (3, 224, 224))

if I train use gray scale image, should I change the 3 into 1 for the above codes?

Thanks！

**Comment by YuzhouPeng** [comment]
yes, if you trained with grayscale image, you need to change this to 1 to fit the channel number, meanwhile you should also change the channel number of input in cpp file's  inference and netconstruction function
